### ISSUE TITLE
Add cluster migration test in cache mode

### ISSUE THREAD

**Issue by chakaz** [dragonflydb/dragonfly#4354]
This last comment makes me think that we are missing cluster migration test for dragonfly running in cache mode and for testing expire logic. Lets open a separate task for that and do this in a separate PR

_Originally posted by @adiholden in https://github.com/dragonflydb/dragonfly/pull/4197#discussion_r1895339014_

**Comment by BorysTheDev** [comment]
@adiholden Could you clarify what stats check we want to verify regarding expiration? Because we use up to 0.5s expiration time in seeder and in the end of migrations process they are all expired

**Comment by adiholden** [comment]
Checkout test_expiry in replication test.
To make sure expire runs at the time of migration run the expire commands only after you start migration.
You will need to check the info stats to see the source node expired some keys to make sure that some keys are expired at the time of migration
